advisories/a88042.md

```diff
@@ -40,11 +40,11 @@ This public issue is tracked by [#87305](https://github.com/cockroachdb/cockroac
 
 ## Mitigation
 
-Users of CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.8](../releases/v22.1.html#v22-1-8) are encouraged to upgrade to [v22.1.9](../releases/v22.1.html#v22-1-9) or a later version.
+Users of CockroachDB [v22.1.0](../releases/v22.1.html#v22-1-0) to [v22.1.8](../releases/v22.1.html#v22-1-8) are encouraged to upgrade to [v21.2.17](../releases/v22.1.html#v21-2-17), [v22.1.9](../releases/v22.1.html#v22-1-9), or a later version.
```

Comment:
This section needs to call out the two major versions. I think this sentence conflates them.